src/assets/apis/config.js

```diff
@@ -1,3 +1,8 @@
-export const MOVIESURL=`http://localhost:8080/api/movies`;
-export const ARTISTSURL=`http://localhost:8080/api/crews`;
-export const  pageRangeDisplayed=3
\ No newline at end of file
+export const BASEURL = `http://localhost:8080/api`;
+export const pageRangeDisplayed = 3;
```

Comment:
It is better to move these constant values where you are using them

Reply:
done